docker-app/qfieldcloud/core/admin.py

```diff
@@ -568,6 +568,20 @@ class ProjectFilesWidget(widgets.Input):
     template_name = "admin/project_files_widget.html"
 
 
+class OwnerTypeFilter(admin.SimpleListFilter):
+    title = _("owner type")
+    parameter_name = "owner_type"
+
+    def lookups(self, request, model_admin):
+        return [(User.Type.PERSON, "Person"), (User.Type.ORGANIZATION, "Organization")]
+
+    def queryset(self, request, queryset):
+        value = self.value()
+        if value is None:
+            return queryset
+        return queryset.filter(owner__type=value)
```

Comment:
```suggestion
        value = self.value()

        if value is None:
            return queryset

        return queryset.filter(owner__type=value)
```
Slight nitpicking, we usually put some air around the `if`/`for`/`try/except` etc blocks :)